.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- Please take a look at our [Contributing](https://rook.io/docs/rook/master/development-flow.html)
+documentation before submitting a Pull Request!
+Thank you for contributing to Rook! -->
+
+Is this a fix for an issue or a feature implementation?
```

Comment:
The question of fix/feature should be sufficiently answered by the description, as well as the link to the issue that is fixed. Can we remove lines 5-8?